### ISSUE TITLE
No icon with thunderbird 91.x

### ISSUE THREAD

**Issue by datafoo** [NixOS/nixpkgs#146478]
### Describe the bug
After migrating from thunderbird 78.14.0 to 91.2.1, the thunderbird icon in the task manager bar was replaced by a generic icon.

### Steps To Reproduce
1. Migrating from thunderbird 78.14.0 to 91.2.1
2. Restart machine

### Expected behavior
The task manager bar icon should remain.

### Screenshots
See the generic icon in the bottom left corner (between KeePassX and Firefox icons):
![Screenshot_20211118_112240](https://user-images.githubusercontent.com/34766150/142397604-85ce3e7c-8055-438f-aafc-4225ba66f4b0.png)

### Notify maintainers
@yu-re-ka @taku0 @r-ryantm 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.79, NixOS, 21.05.4116.46251a79f75 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(myuser): `"home-manager-21.05"`
 - channels(root): `"nixos-21.05.4116.46251a79f75"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixos modules affected by the problem
module: thunderbird
```

**Comment by taku0** [comment]
Maybe related to https://github.com/NixOS/nixpkgs/pull/145858.

**Comment by OPNA2608** [comment]
Fixed on my 21.05 system now, although I had to compile thunderbird from source ([#146719](https://github.com/NixOS/nixpkgs/issues/146719#issuecomment-974567569)).

**Comment by vcunat** [comment]
All solved, I believe.
